release.json

```diff
@@ -44,34 +44,34 @@
         "WINDOWS_APMINJECT_SHASUM": "5fdd62a84e640204386b9c28dc2e3ac5d9b8adde6427cb9f5914619f94d7b5bd"
     },
     "release-a6": {
-        "INTEGRATIONS_CORE_VERSION": "7.56.0-rc.2",
-        "OMNIBUS_SOFTWARE_VERSION": "7.56.0-rc.3",
-        "OMNIBUS_RUBY_VERSION": "7.56.0-rc.1",
+        "INTEGRATIONS_CORE_VERSION": "7.58.0-rc.1",
+        "OMNIBUS_SOFTWARE_VERSION": "7.58.0-rc.1",
+        "OMNIBUS_RUBY_VERSION": "7.58.0-rc.1",
         "JMXFETCH_VERSION": "0.49.3",
         "JMXFETCH_HASH": "258085a94d529a6bdf914db36dd50faf6fde2cebc44b1f54a60eb209a5d8917c",
         "SECURITY_AGENT_POLICIES_VERSION": "v0.58.0",
-        "MACOS_BUILD_VERSION": "6.56.0-rc.3",
+        "MACOS_BUILD_VERSION": "7.58.0-rc.1",
```

Comment:
Hmm - shouldn't this be `  "MACOS_BUILD_VERSION": "6.58.0-rc.1"`?

Reply:
Nice catch, correct I forgot to do the `6.58.0-rc.1` tag on macos. It's done now and I've updated the value